README.adoc

```diff
@@ -55,6 +57,12 @@ This project has dependency and transitive dependencies on Spring Projects. The
 |===
 | Spring Framework on Google Cloud | Spring Cloud | Spring Boot | Spring Framework | Supported
 
+|7.x
+|https://github.com/spring-cloud/spring-cloud-release/wiki/Spring-Cloud-2025.0-Release-Notes[2025.0.x] (Northfields)
+|https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.5-Release-Notes[3.5.x]
+|https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-6.2-Release-Notes[6.2.x]
```

Comment:
nit: is Spring Boot 3.5.x compatible with Spring Boot Framework 6.2.x same as 3.4.x? or is it 6.2.7+?